src/ground.rs

```diff
@@ -0,0 +1,26 @@
+use crate::cartesian::XZPoint;
+
+pub struct Ground {
+    //
+}
+
+impl Ground {
```

Comment:
This is where the sinewave comes from. When we start using actual elevation data this is where it would go